doc/examples/controller_configuration/controller_configuration_tutorial.rst

```diff
@@ -1,53 +1,35 @@
+
+#. Launch the corresponding :code:`ros2_control` JointTrajectoryControllers. This is separate from the MoveIt2 ecosystem. `Example ros2_control launching <https://github.com/ros-controls/ros2_control_demos>`_. Each JointTrajectoryController provides an action interface. Given the yaml file above, MoveIt automatically connects to this action interface.
+
+#. Note: it is not required to use :code:`ros2_control` for your robot. You could write a proprietary action interface. In practice, 99% of users choose :code:`ros2_control`.
+
+MoveIt Controller Manager
+-------------------------
+If using the Move Group, a MoveItControllerManager (MICM) can be used to manage controller switching. The MICM can parse the joint names in any command coming from MoveIt and activate the appropriate controllers. For example, it can automatically switch between controlling two manipulators in a single joint group at once to a single manipulator. To use a MICM, just set :code:`moveit_manage_controllers = true` in the launch file. `Example MICM launch file <https://github.com/ros-planning/moveit_resources/blob/ros2/panda_moveit_config/launch/demo.launch.py>`_. Frankly, the MICM is a candidate to be deprecated soon and we do not recommend its use.
+
+MoveIt Controller Interfaces
+----------------------------
+
+The text above describes launching of a joint trajectory controller action interface. In addition, MoveIt supports gripper control. This section describes the parameters of these two options.
+
+*. FollowJointTrajectory Controller Interface
 
- controller_list:
-  - name: panda_arm_controller
-    action_ns: follow_joint_trajectory
-    type: FollowJointTrajectory
-    default: true
-    joints:
-      - panda_joint1
-      - panda_joint2
-      - panda_joint3
-      - panda_joint4
-      - panda_joint5
-      - panda_joint6
-      - panda_joint7
-  - name: hand_controller
-    action_ns: gripper_action
-    type: GripperCommand
-    default: true
-    parallel: true
-    joints:
-      - panda_finger_joint1
-      - panda_finger_joint2
-
-There are many different parameters that can be used for different types of controllers.
-
-FollowJointTrajectory Controller Interface
-------------------------------------------
 The parameters are:
  * *name*: The name of the controller.  (See debugging information below for important notes).
  * *action_ns*: The action namespace for the controller. (See debugging information below for important notes).
  * *type*: The type of action being used (here FollowJointTrajectory).
  * *default*: The default controller is the primary controller chosen by MoveIt for communicating with a particular set of joints.
  * *joints*: Names of all the joints that are being addressed by this interface.
 
-GripperCommand Controller Interface
------------------------------------
+*. GripperCommand Controller Interface
```

Comment:
I'm fuzzy on the requirements for a GripperCommand controller. I think it's for parallel-jaw grippers only? I lean toward deprecating it.